### DISCUSSION TITLE
Download ends before entire file downloaded.

### DISCUSSION THREAD

**Discussion by CrashM** [aclap-dev/video-downloadhelper#96] [Help and Q&A]
### Your question

I'm fairly new to the app which I've also paid for.
Seemed to be working much better before I bought the license.  Hope that's a coincidence.
I've attempted to download different files from this site (and other related), all show the same issue of ending early.

Attempting to download from this site: 
https://www.primewire.tf/tv/1483380/payback-season-1-episode-6
Hit details indicates URL:
https://a-delivery27.mxdcontent.net/v/8ee62c2f6136259850db077b61ad64eb.mp4?s=QRdbW3R9WMvxqtveDli1sg&e=1707984372&_t=1707969761

Application info:
version: 8.2.0.22
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: edge
lang: en-US
license: accepted
key (only 16 first characters): 9da06fe092d14859
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\xxxxx",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
It's a little hard to reproduce because that website has different servers, with different technologies.

- Can you be more specific on which video you're attempting to download?
- Can you share the error message if any?
- Do you mind [updating your coapp](https://www.downloadhelper.net/w/CoApp-Installation)?

**Reply by paulrouget** [reply]
Also, I need to know, in the settings, in behavior, which download method you're using.

**Comment by CrashM** [comment]
On the original question, there were no error messages that I could see,
just the video download terminating prematurely.

However, over the last couple days, this has not occured again, don't know
if that's due to a newer version of your software, or a temporary network
issue of some sort.

Lets close this case, and I'll monitor and start a new thread if
warrented.  Many thanks...

On Fri, Feb 16, 2024 at 11:51 AM Paul ***@***.***> wrote:

> Also, I need to know, in the settings, in behavior, which download method
> you're using.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aclap-dev/video-downloadhelper/discussions/96#discussioncomment-8494956>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACXRNMY3CDWIWU44OC3D363YT6FAJAVCNFSM6AAAAABDJOY632VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIOJUHE2TM>
> .
> You are receiving this because you authored the thread.Message ID:
> <aclap-dev/video-downloadhelper/repo-discussions/96/comments/8494956@
> github.com>
>


-- 

Regards,
Mike Rilling

**Comment by paulrouget** [comment]
Wait a little bit, we have an upcoming version 8.2.1 which might solve that specific issue.

Once it updated (should happen today or tomorrow), please give it another try, and let me know how it went!

**Comment by paulrouget** [comment]
I think this will fix this issue: https://github.com/aclap-dev/video-downloadhelper/discussions/236
